openedx/core/djangoapps/content/learning_sequences/data.py

```diff
@@ -25,17 +25,23 @@
 """
 import logging
 from datetime import datetime, timezone
+from enum import Enum
 from typing import Dict, List, Optional, Set
 
 import attr
 from django.contrib.auth import get_user_model
 from opaque_keys.edx.keys import CourseKey, UsageKey
 
-
 User = get_user_model()
 log = logging.getLogger(__name__)
 
 
+class CourseVisibilities(Enum):
```

Comment:
Generally, class names should be in singular form, not plural. So I think this should probably be renamed to `CourseVisibility`

Reply:
Yep, good catch

Reply:
@pcockwell Fixed

Reply:
Thanks!